tests/sentry/models/test_userrole.py

```diff
@@ -0,0 +1,14 @@
+from sentry.models import UserRole
+from sentry.testutils import TestCase
+
+
+class UserRoleTest(TestCase):
+    def test_permissions_for_user(self):
+        user = self.create_user(email="a@example.com")
+        user2 = self.create_user(email="b@example.com")
+        role = UserRole.objects.create(name="test", permissions=["test1", "test2"])
+        role.users.add(user)
+        role2 = UserRole.objects.create(name="test", permissions=["test2", "test3"])
```

Comment:
Should roles have unique names? Will be confusing if there are two roles with the same display names.

Reply:
oops just bad test (i hadnt run it locally)